docs/UserGuide.md

```diff
@@ -0,0 +1,23 @@
+# Mismatch Finder User Guide
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Logging in <a id="login"></a>](#logging-in-a-idlogina)
+- [Obtaining an API token <a id="apiToken"></a>](#obtaining-an-api-token-a-idapitokena)
+- [Accessing the API <a id="apiAccess"></a>](#accessing-the-api-a-idapiaccessa)
```

Comment:
I'm pretty sure these `<a>` tags aren't supposed to be here. We also need to make sure that the `#` link is correct. See suggestion.

```suggestion
- [Logging in](#login)
- [Obtaining an API token](#apiToken)
- [Accessing the API](#apiAccess)
```